### ISSUE TITLE
Correlation tables confidence interval and M SD columns

### ISSUE THREAD

**Issue by acescos** [dstanley4/apaTables#9]
Hi David and Jeffrey,
I am very happy to use your R apaTables package. 
It would be very useful to me that apaTables present the options for including or not (a) the confidence intervals and, (b) Mean and standard deviation columns in the correlation matrix.
Best
Ale

**Comment by DanSimonet** [comment]
I concur with this comment. I do appreciate the movement towards representing uncertainty in our data, but also note with appropriate descriptive information it is possible to create confidence intervals directly (for readers). In certain cases I often desire to create large correlation matrices which also fit onto a single page. In such instances it would be desirable to retain the feature to hide confidence intervals when producing correlation matrices.

**Comment by sam-cannon** [comment]
I agree, having the confidence interval be a True/False argument would be really helpful

**Comment by acounsell** [comment]
I am late to using the apaTables package but have a suggestion related to those discussed above. While I agree with the comments about making these features optional, I think folks would find it particularly useful if there was also an argument to change the orientation of the M, SD columns so that they are included as rows at the bottom of the table instead? This feature would be a big help when we have page length than width. Thanks for this awesome package!
